src/v/kafka/protocol/CMakeLists.txt

```diff
@@ -4,7 +4,16 @@ v_cc_library(
   NAME kafka_protocol
   SRCS
     errors.cc
+    ${schemata_srcs}
+  COPTS
+    "-Wno-unused-lambda-capture"
   DEPS
-    v::kafka_request_schemata)
+    Seastar::seastar
+    v::bytes
+    v::kafka_protocol
```

Comment:
did you intend to remove `v::kafka_protocol` here -- seems like a circular reference

Reply:
ahh, very true. thanks!

fixed and repushed.